### PR TITLE
feat(assets): extract all assets into a folder

### PR DESCRIPTION
**Pull request by Gabriel-Ladzaretti**
context 
 - https://github.com/renovatebot/tutorial/pull/28#discussion_r1282399158

### CODE REVIEW

**Comment by Gabriel-Ladzaretti** [comment]
@HonkingGoose

File: .gitignore [hunk 1]
```diff
@@ -5,6 +5,7 @@ npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
 lerna-debug.log*
+.idea
```
Comment (HonkingGoose) [inline .gitignore:8]: GitHub has a repository with good gitignore templates. They have a [`.gitignore` template for Jetbrains IDE's](https://github.com/github/gitignore/blob/main/Global/JetBrains.gitignore).

Maybe we should copy/paste some things from them? Before copying, check if their license is okay for us: [`github/gitignore` license file](https://github.com/github/gitignore/blob/main/LICENSE).

File: assets/change-in-config-1.png
Comment (HonkingGoose) [inline assets/change-in-config-1.png:1]: We should use plain JSON text instead of a screenshot.

File: assets/pending-approval-config.png
Comment (HonkingGoose) [inline assets/pending-approval-config.png:1]: This screenshot should be converted to plain text JSON.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Pull request closed without merging**